packages/experiments/src/components/Chiclet/ChicletXsmall.styles.tsx

```diff
@@ -81,6 +83,9 @@ export const getStyles = (styleProps: IChicletCardStyleProps): IChicletCardStyle
         // @todo: change values for height and width according to redline
```

Comment:
Do we need the todo anymore? Are height and width provided in the redlines?

Reply:
Same question for the icon styling above

Reply:
No, I will delete the todos that are complete!